third_party/BUILD

```diff
@@ -623,6 +623,22 @@ filegroup(
     ],
 )
 
+genrule(
+    name = "java_tools_dist",
+    srcs = [
+        "//src/java_tools/buildjar:srcs",
+        "//src/java_tools/junitrunner:srcs",
+        "//src/java_tools/singlejar:srcs",
+        "//third_party/jarjar:srcs",
+        "//third_party/java/jdk/langtools:license-and-srcs",
+        "//third_party/zlib:embedded_tools",
+        "//third_party/ijar:srcs_with_deps",
+    ],
+    outs = ["java_tools_dist.zip"],
+    cmd = "zip -Xr $@ $$(echo $(SRCS) | sort)",
+    output_to_bindir = 1,
```

Comment:
Why is this necessary?

Reply:
Hm just for consistency with the other java_tools_zip rule... It's not a hard requirement.